metagov/metagov/core/utils.py

```diff
@@ -112,22 +112,17 @@ def get_process_schemas(cls):
 
 def validate_and_fill_defaults(values, schema):
     from metagov.core.validators import DefaultValidatingDraft7Validator
-    from rest_framework.exceptions import ValidationError
-
-    try:
-        # this mutates `plugin_config` by filling in default values from schema
-        DefaultValidatingDraft7Validator(schema).validate(values)
-    except jsonschema.exceptions.ValidationError as err:
-        raise ValidationError(f"ValidationError: {err.message}")
+    # this mutates `plugin_config` by filling in default values from schema
+    # raises jsonschema.exceptions.ValidationError
+    DefaultValidatingDraft7Validator(schema).validate(values)
```

Comment:
Why are we moving this out of a try/except block? I trust you, just curious. :)

Reply:
I don't think it makes sense for the `MetagovApp` to raising `ValidationError`s, since ValidationErrors are from the django rest framework.. we definitely don't have a clean separation here (yet) but it would be nice if the internal API does not depend on DRF. I just changed it here so we catch the `jsonschema.exceptions.ValidationError` in the view, and raise a `ValidationError` there.

I think a further improvement would probably be to raise a custom `MetagovError` here, and have the internal API expose it (Along with several others, like PluginNotEnabled, CommunityNotFound, etc) so that Django Drivers can catch those exceptions, instead catching the root exceptions like `Community.DoesNotExist` etc.